components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -41,9 +41,13 @@
         <div class="header">
           <div class ="service-line"><chef-icon>storage</chef-icon><b>{{ service.fqdn }}</b></div>
           <div class ="supervisor-line"><b>Supervisor:</b> {{ service.supervisor_id }}</div>
+          <div class ="supervisor-line" *ngIf="service.channel"><b>Site:</b> {{ service.site }}</div>
+          <div class ="supervisor-line" *ngIf="service.channel == ''"><b>Site:</b> --</div>
         </div>
         <div class="body">
           <div class ="service-line"><chef-icon>grain</chef-icon>{{ service.release }}</div>
+          <div class ="service-line" *ngIf="service.channel"><chef-pill>{{ service.channel }}</chef-pill></div>
+          <div class ="channel-line" *ngIf="service.channel == ''"><chef-pill> - - </chef-pill></div>
```

Comment:
I can see we are always displaying the `Site` and the `Channel`, even when they have no values from the backend. I would recommend writing a function to do this logic in the `.ts` file and not in the HTML with `*ngIf`. That way you could even write a simple test to ensure they return the expected value. 😄 
(I say that because one returns `--`and the other one `- -`)

Reply:
I really only added the space because it looks better but maybe that is something we can fix with css?